approve.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+
+set -o errexit;
+set -o nounset;
+
+cd examples
+
+if [ ! -f elm.js ]; then
+    printf "Missing elm.js, run \`npm run build-examples\` to generate\n"
+    exit 1
+fi
```

Comment:
```suggestion
npm run build-examples
```

Any reason not to do this?

Reply:
Slow? Although probably not too bad

Reply:
I'm not sure I'm following what would make it slow, isn't it going to run it regardless? We can remove the `npm run build-examples` from the inline script in travis, and have it compile and run the examples in `approve.sh`.

Either way, though, robustness and foolproofing seems like an essential to me. I'm not sure the speed would be noticeable (unless I'm missing something).

Reply:
Agreed.